### PR TITLE
Add fbo complete test for fbo attaching base level of a texture

### PR DESCRIPTION
**Pull request by qkmiao**
This failed on NVIDIA Linux with ANGLE and Intel Mac OSX. Chrome bug is filed: https://bugs.chromium.org/p/chromium/issues/detail?id=678526.

### CODE REVIEW

**Comment by kainino0x** [comment]
LGTM

File: sdk/tests/conformance2/renderbuffers/00_test_list.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+--min-version 2.0.1 framebuffer-completeness.html
```
Comment (zhenyao) [inline sdk/tests/conformance2/renderbuffers/00_test_list.txt]: Can we change the name to be more specific, like framebuffer-texture-with-non-zero-base-level.html, something like that?

Comment (qkmiao) [inline sdk/tests/conformance2/renderbuffers/00_test_list.txt]: We can add framebuffer completeness related tests in this file and use different function names like runBaseLevelTest to differentiate them.  Does this make sense? If not, I can change the file name.

Comment (zhenyao) [inline sdk/tests/conformance2/renderbuffers/00_test_list.txt]: Then framebuffer-test.html sounds like a perfect place for this test case.

Comment (qkmiao) [inline sdk/tests/conformance2/renderbuffers/00_test_list.txt]: Move the test to framebuffer-test.html

**Comment by zhenyao** [comment]
Thank you. LGTM

**Pull request merged**